listener/advisor_processor.py

```diff
@@ -208,6 +209,9 @@ async def _process_upload(self, msg: AdvisorMsg):
                         return
                 # Insert new items to the cache only after succesfully commited insert/update
                 await self._commit_operating_system_cache(operating_system, operating_system_id)
+            except psycopg_errors.UntranslatableCharacter as exc:
+                LOGGER.exception("Ignoring system with corrupt data (null bytes): %s, org_id: %s, error: %s", inventory_id, org_id, exc)
```

Comment:
I think using `LOGGER.warning()` would be better because `.exception()` is error level + prints a traceback which is not needed here.

And I'd not specifically mention null bytes, it can be something different as well